packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx

```diff
@@ -47,38 +48,46 @@ export default function SwapSettingDialog({ open, onClose }: SettingDialogProps)
 
     const xDaiOptions = [{ label: 'SushiSwap', value: TradeProvider.SUSHISWAP }]
 
+    const celoOptions = [{ label: 'SushiSwap', value: TradeProvider.SUSHISWAP }]
+
     const t = useDashboardI18N()
 
     const items = [
         {
             legend: t.labs_settings_swap_network({ network: 'ETH' }),
             value: useEthereumNetworkTradeProvider(),
             options: ethOptions,
-            onChange: (value: any) => Services.Settings.setEthNetworkTradeProvider(+value),
+            onChange: (value: string) => Services.Settings.setEthNetworkTradeProvider(Number(value)),
```

Comment:
plz use `Number.parseInt` function, thanks

https://mdn.io/number-parseint

Reply:
@septs Done :)

Reply:
@septs Could you setup an eslint to check that?

Maybe this rule: https://eslint.org/docs/rules/no-restricted-globals